sdk/storage/azure-storage-blob-batch/pom.xml

```diff
@@ -55,7 +55,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.11.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
+      <version>1.12.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
```

Comment:
```suggestion
      <version>1.12.0-beta.1</version> <!-- {x-version-update;unreleased_com.azure:azure-core;dependency} -->
```

Reply:
This will be the version tag to use until the next version of azure-core is released. A new line for this may need to be added to the version text file

Reply:
I'm thinking about keeping this PR open till core goes out. I don't want to block ability to release storage sdk.